app/client/__tests__/components/identity/OptOutSection.tsx

```diff
@@ -0,0 +1,53 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import React from "react";
+import { OptOutSection } from "../../../components/identity/EmailAndMarketing/OptOutSection";
+import {
+  ConsentOption,
+  ConsentOptionType,
+} from "../../../components/identity/models";
+
+afterEach(cleanup);
+
+const mockClickHandler = jest.fn();
+
+describe("OptOutSection component", () => {
+  it("correctly displays toggle switch values for OPT OUT consent types", () => {
+    render(
+      <OptOutSection
+        consents={idapiConsentsFixture}
+        clickHandler={mockClickHandler}
+      />
+    );
+
+    expect(screen.getByLabelText("Allow post")).toHaveAttribute(
+      "aria-checked",
+      "true"
+    );
+    expect(screen.getByLabelText("Allow phone")).toHaveAttribute(
+      "aria-checked",
+      "false"
+    );
```

Comment:
https://github.com/testing-library/jest-dom#tobechecked

`@testing-library/jest-dom` should probably be set up in all these tests, currently it seems to have been manually included in a few.

Reply:
Yes - i wasn't sure how much to mess around with other tests. Will look at raising a seperate pr to change testing. I spent some time to try using .toBeChecked attribute but it doesn't seem to work based on the aria role=switch even though according to the docs it should. I will get it set up again and maybe you could help me debug what I was doing wrong or maybe there is a bug

Reply:
I had a  look and setting this up in a jest config file and it broke some global imports so would prefer to look at in a separate PR so this one doesn't get unwieldy.